package.json

```diff
@@ -66,6 +65,7 @@
     "write-file-webpack-plugin": "3.4.2"
   },
   "dependencies": {
+    "@types/d3": "4.5.0",
```

Comment:
This now needs to be a semver range so that users can pick up fixes to typings. You can lock it down in plottable development with yarn.lock